generators/app/templates/app/middlewares/apiInfo.js

```diff
@@ -0,0 +1,15 @@
+const pjson = require('../../package.json');
+
+exports.apiInformation = (req, res, next) => {
+  try {
+    if (!res.headers) {
+      res.headers = {};
+    }
+    res.headers['X-API-Date'] = new Date();
+    res.headers['X-Package-Version'] = pjson.version;
+    res.headers['X-Node-Version'] = pjson.engines.node;
```

Comment:
Should be nice add a configuration in config.js for headers name, something like:
```
res.headers[config.common...] = new Date();
...
```